### PR TITLE
updating FEB most probable, and url for data

### PR DESCRIPTION
**Pull request by jread-usgs**

### CODE REVIEW

**Comment by jread-usgs** [comment]
@jiwalker-usgs can you review?

**Comment by jread-usgs** [comment]
note @eread-usgs that the "Most probable" text no longer includes the month of the estimate, as that isn't in the data column anymore:
image

File: scripts/R/manipulate_mead_projected.R [hunk 1]
```diff
@@ -251,20 +251,64 @@ add_lines <- function(g_id, data, form.factor, language){
...
+
+read_mead_historical <- function(){
   
+  data = read.csv('src_data/DynamicMeadHistorical.csv', stringsAsFactors = F)
+  data$posDate = as.POSIXct(data$posDate, format='%Y-%m-%d')
+  return(data)
+}
+
+get_mead_filename <- function(){
+  date <- Sys.time()-86400*14 # two weeks earlier
```
Comment (jiwalker-usgs) [inline scripts/R/manipulate_mead_projected.R]: Couldn't this be used to recreate the "Most Probable" month, or is that a stretch on our part?

Comment (jiwalker-usgs) [inline scripts/R/manipulate_mead_projected.R]: Also, this forces us to run the job within the first two weeks of a month to get the previous month.  Is that the right assumption?

Comment (jread-usgs) [inline scripts/R/manipulate_mead_projected.R]: I wasn't sure, hence leaving out the labeling and keeping the `get_mead_filename()` as a function that we can easily change if there is some better logic we can use.

**Pull request merged**